frontend/server/src/test/resources/metrics_default.yaml

```diff
@@ -0,0 +1,35 @@
+dimensions:
+  ModelName: "my_tc"  # customer should edit based on their own models that are being used
```

Comment:
Right now can we have a per model metric config? My understanding was no, in which case we shouldnt have a ModelName field